test/e2e/runner/start.go

```diff
@@ -66,8 +69,22 @@ func Start(testnet *e2e.Testnet) error {
 		}
 	}
 
-	// Start up remaining nodes
 	for _, node := range nodeQueue {
+		// if we're starting a node in the "future" relative
+		// to where the network currently is, we should make
+		// sure the network has gotten to that point first.
+		if node.StartAt > initialHieght {
+			logger.Info("Waiting for network to advance before starting catch up node",
+				"node", node.Name,
+				"height", node.StartAt)
+
+			if _, _, err := waitForHeight(testnet, node.StartAt); err != nil {
+				return err
+			}
+
+			initialHieght = node.StartAt
```

Comment:
Do we really need the _max_, or is any height in bound OK?

Reply:
I think the logic is correct, but I changed the variable names